ouroboros-network/demo/chain-sync.hs

```diff
@@ -101,10 +107,17 @@ mkLocalSocketAddrInfo socketPath =
       (Socket.SockAddrUnix socketPath)
       Nothing
 
+defaultLocalSocketAddrPath :: FilePath
+defaultLocalSocketAddrPath =  "./demo-chain-sync.sock"
+
 defaultLocalSocketAddrInfo :: Socket.AddrInfo
 defaultLocalSocketAddrInfo = 
-    mkLocalSocketAddrInfo "./demo-chain-sync.sock"
+    mkLocalSocketAddrInfo defaultLocalSocketAddrPath
 
+rmDefaultLocalSocketAddr :: IO ()
+rmDefaultLocalSocketAddr = do
+  b <- doesFileExist defaultLocalSocketAddrPath
+  when b (removeFile defaultLocalSocketAddrPath)
```

Comment:
It'd probably be more convenient to delete the socket in all the server cases, not just the one where the name is the default one. Not a big deal.

Reply:
Right; fixed.